lo.h

```diff
@@ -5,6 +5,7 @@
 #include <map>
 #include <v8-fast-api-calls.h>
```

Comment:
there's a bigger question about how we can import this in bindings and in general how we will handle "cross-runtime" bindings. i will create an issue for it.